### PR TITLE
User Profile

### PR DESCRIPTION
**Pull request by shirishgoyal**
image

image

Pending City and Country Migrations

### CODE REVIEW

**Comment by shirishgoyal** [comment]
@mbernst Do you have list of cities and countries we can allow at this stage? In other case, we can also use https://developers.google.com/places/javascript/ to fetch all information related to address.

**Comment by mbernst** [comment]
I agree, let's go for any standardized set of places. Google's set — maybe
even using their widgets — doesn't seem like a terrible idea.

**Comment by thunderboltsid** [comment]
Google's standard set looks good. Can you user their dataset without using their map visualisation? Or is the map something we want?

**Comment by shirishgoyal** [comment]
@thunderboltsid I am specifically referring to this demo : https://developers.google.com/maps/documentation/javascript/examples/places-autocomplete-addressform

But after more discussion, data migration makes more sense I think since we might be linking city and country at other places too in future.

**Comment by dmorina** [comment]
@thunderboltsid @shirishgoyal how are we doing on this one?

**Comment by qwertyone** [comment]
Bad option: So I found a data set that is updated weekly, but it's a bad idea since the raw files are extremely large. ftp://ftp.nga.mil/pub2/gns_data/ this is posted to enable comparison of alternatives, and this is the worst.

**Comment by shirishgoyal** [comment]
For now, have integrated google places API to fill in address. 
City / Country are not linked.

**Comment by dmorina** [comment]
move view/edit profile into the More_Vert drop down

**Comment by dmorina** [comment]
some of the drop downs are too wide e.g gender and ethnicity

File: crowdsourcing/viewsets/user.py [hunk 1]
```diff
@@ -11,10 +11,11 @@
 from crowdsourcing.redis import RedisProvider
 from crowdsourcing.serializers.user import UserProfileSerializer, UserSerializer, UserPreferencesSerializer
 from crowdsourcing.permissions.user import CanCreateAccount
+from crowdsourcing.serializers.utils import CountrySerializer, CitySerializer
 from crowdsourcing.utils import get_model_or_none
 
 
-class UserViewSet(mixins.RetrieveModelMixin, mixins.CreateModelMixin, viewsets.GenericViewSet):
+class UserViewSet(viewsets.ModelViewSet):
```
Comment (dmorina) [inline crowdsourcing/viewsets/user.py]: i dont think we wanna allow all 5 methods by default, just add mixins.UpdateModelMixin

**Comment by dmorina** [comment]
screen shot 2016-04-27 at 3 13 37 pm
could we have no defaults here?

otherwise lgtm, see comment

**Pull request merged**